.github/workflows/velox_be.yml

```diff
@@ -44,33 +44,17 @@ concurrency:
 
 jobs:
   velox-ubuntu2004-test-spark32:
-    runs-on: velox-self-hosted
+    runs-on: velox-self-hosted-image
```

Comment:
Question: what's the difference between ```velox-self-hosted-image``` and ```velox-self-hosted```? Is it on K8S?

Reply:
not in K8S, velox-self-hosted-image can pull image from private docker registry